analyses/country_template/docs/template.Rmd

```diff
@@ -0,0 +1,51 @@
+---
+title: '<img src="https://drive.google.com/uc?id=118y5T73-iSUZoAMtoJGddxq9QzD_GDKX" style="height:40px;float:left; margin:10px" /><img src="https://drive.google.com/uc?id=1fHQUzF3ZjaoHj9KQ33-94dK_X1hcmjzW" style="height:50px;float:left; margin:10px" />'
+pagetitle: 'Docs Template'
```

Comment:
Should we include comments to call out the lines in the header that need to be updated when the template is used?

Reply:
Yes, good idea